posts/2020-10-23-release-post-200011.adoc

```diff
@@ -0,0 +1,143 @@
+
+The `kerberos` configuration element in `server.xml` provides system-wide configuration options for the Liberty server. For example:
+
+[source, xml]
+----
+  <kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf"/>
+----
+
+Below is an example of the Kerberos protocol being used to secure a data source: 
+
+[source, xml]
+----
+  <featureManager>
+    <feature>jdbc-4.2</feature>
+  </featureManager>
+
+  <!-- optional config: This is only needed if you need to customize the location of keytab or krb5.conf -->
+  <kerberos keytab="${server.config.dir}/security/krb5.keytab" configFile="${server.config.dir}/security/krb5.conf"/>
+
+  <authData id="myKerberosAuth" krb5Principal="krbUser"/>
+
+  <library id="db2DriverLib">
+    <fileset dir="${server.config.dir}/db2"/>
+  </library>
+
+  <dataSource jndiName="jdbc/krb/basic" containerAuthDataRef="myKerberosAuth">
+    <jdbcDriver libraryRef="db2DriverLib"/>
+    <properties.db2.jcc databaseName="${DB2_DBNAME}" serverName="${DB2_HOSTNAME}" portNumber="${DB2_PORT}"/>
+  </dataSource>
+----
+
+Before this release, it was technically possible to use Kerberos authentication with JDBC data sources, but the configuration was complex and un-documented. Also, was a lack of connection pooling support when using Kerberos authentication for data sources.
+
+////
+Full documentation for this capability will be available soon in the link:{url-prefix}/docs/latest/overview.html[Open Liberty documentation] pages.
+////
+
+[#grafana]
+== Grafana dashboard support for Thanos data source
+
+With Open Liberty 20.0.0.11 the Open Liberty Grafana dashboard, which visualizes data from the MicroProfile Metrics feature, is now able to visualize data from a Thanos data source.
+
+The Grafana dashboard provides a wide range of time-series visualizations of MicroProfile Metrics data such as CPU, Servlet, Connection Pool, and Garbage Collection metrics. It is powered by a Prometheus data source which is configured to ingest data from the `/metrics` endpoint of one or more Liberty servers, enabling you to view performance metrics in near real-time.
+
+Previously, support has only been provided for visualizing metrics data with Grafana on Open Liberty servers that used Prometheus as the data source. In Kubernetes environments, such as the Red Hat OpenShift Container Platform, you can use Thanos to query and store metrics data from multiple clusters. The previous Grafana Dashboards would not work when Thanos was set as the data source, whereas the new Grafana dashboard allows users using Thanos as the data source to display metrics data.
+
+[.img_border_dark]
+image::img/blog/grafana.png[align="center",Image of the updated Grafana dashboard from 20.0.0.11.]
+
+For more information:
+
+link:https://github.com/OpenLiberty/open-liberty-operator/tree/master/deploy/dashboards/metrics/[Open Liberty Grafana dashboards]
```

Comment:
Can you make these three links into a bulleted list (makes it more compact and associated)?

Reply:
Changed.